lib/model/rofolder.go

```diff
@@ -56,10 +46,10 @@ func (f *roFolder) Serve() {
 		case <-f.stop:
 			return
 
-		case <-f.scan.timer.C:
+		case <-f.scan.Timer().C:
```

Comment:
Same here.

perhaps `<-f.scan.Request()`  or something along the lines?

```
func (s *scan) Request() chan time.Time {
  return s.timer.C
}
```
No?